src/main/java/steps/StepDefinitions.java

```diff
@@ -0,0 +1,30 @@
+package steps;
+
+import io.appium.java_client.AppiumDriver;
+import pages.LoginPage;
+import pages.MainPage;
+
+public class StepDefinitions {
+    private AppiumDriver driver;
+    LoginPage loginPage;
+    MainPage mainPage;
+
+    public StepDefinitions(AppiumDriver driver) {
+        this.driver = driver;
+        loginPage = new LoginPage(driver);
+        mainPage = new MainPage(driver);
+    }
+
+    public void loginAsExistingUser() {
+        loginPage.allowNotifications()
+                .clickAlreadyHaveAccount()
+                .clickContinueWithEmail()
+                .fillEmailField("yazio_test@yopmail.com")
```

Comment:
Please don't hardcode credentials inside POs. Even if it's a known user there might be several of them, so it's better to pass them as an argument so that the method is reusable. I'd recommend to create UserFactory class to manage different users